### DISCUSSION TITLE
make painter work in archive ?

### DISCUSSION THREAD

**Discussion by hellersqueen** [AprilSylph/XKit-Rewritten#1411] [Suggestion Box]
i dont know if that's something that could be possible to make painter work both on the dashboard and in the archive of someone's blog?

**Comment by AprilSylph** [comment]
The Archive doesn't exist under `www.tumblr.com`, and so the session there is never truly logged-in, which severely limits what anything in XKit can do to it. For Painter, this would mean the "Own post colour" and "Liked post colour" options never working (the latter of which seems the most useful for the Archive IMO).

I think it'd be more confusing to have the feature only work partially in the Archive than to have it not work at all, as it does currently.

**Reply by hellersqueen** [reply]
oh yeah the "liked post colour" was the one i really wanted to work actually  :/  thanks for the answer !
